vm/vm.go

```diff
@@ -211,7 +211,7 @@ func getMapIndex(key reflect.Value, aMap reflect.Value) reflect.Value {
 			keyType = key.Elem().Type()
 		}
 	}
-	if keyType != aMap.Type().Key() {
+	if keyType != aMap.Type().Key() && aMap.Type().Key() != interfaceType {
```

Comment:
This should change use convertReflectValueToType at some point.

Reply:
I can't move test from *TestMakeMapsData()* to *TestMaps()* because if we set map in an Anko script, keys would be *string*, not *interface{}*.

Reply:
Added more unit tests.